### ISSUE TITLE
Feature Request: Add bar to change trasnparency

### ISSUE THREAD

**Issue by AB10110F** [nvzone/minty#16]
In my opinion that's the only thing that this plugin is missing.
Great job with the tui btw

**Comment by siduck** [comment]
show an example, isnt transparency just lightening/darkening?

**Comment by AB10110F** [comment]
> show an example, isnt transparency just lightening/darkening?

In hex colors it is represented adding two more characters to the hex code

Here I have two bars both have the same color but the second one has transparency
.nv {
  ...
  background: #ecb3e5;
}

.nv2 {
  background: #ecb3e560;
}

![image](https://github.com/user-attachments/assets/d39c18b2-5024-48b7-82a5-29565e6c44c9)

**Comment by siduck** [comment]
thats just css related, no? i dont think that hexcolor is applicable everywhere :thinking:

**Comment by AB10110F** [comment]
> thats just css related, no? i dont think that hexcolor is applicable everywhere 🤔

Indeed, I don't think you can use it everywhere, I only use it when working with CSS

**Comment by siduck** [comment]
i dont think its possible to show that color in neovim

**Comment by siduck** [comment]
but i'll see what i can do, after a month
